CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 #### New features and UX changes
 
+- Add `--user`/`--group` option to rkt run/prepare ([#2419](https://github.com/coreos/rkt/pull/2419)). This option allows overriding the user/group specified in the image manifest.
```

Comment:
We're not really consistent on this... https://github.com/coreos/rkt/blob/master/Documentation/commands.md#global-options